amp/repository/aim/view/teamAuditList.jsp

```diff
@@ -25,7 +25,91 @@ function compareAll(){
 	document.aimCompareActivityVersionsForm.submit();	
 	}
 
+function toggleFilterSettings(){
+	var currentFilterSettings = $('#currentFilterSettings');
+	var displayFilterButton = $('#displayFilterButton');
+	if(currentFilterSettings.css('display') == "inline-flex"){
+		currentFilterSettings.hide();
+		$('#exportScorecard').hide();
+		displayFilterButton.html('<digi:trn jsFriendly="true" key="aim:Showfilteroptions">Show Filter options</digi:trn>'+ ' &gt;&gt;');
+	}
+	else
+	{
+		currentFilterSettings.css('display', 'inline-flex');
+		$('#exportScorecard').css('display','inline-flex');
+		displayFilterButton.html('<digi:trn jsFriendly="true" key="aim:Hidefilteroptions">Hide Filter options</digi:trn>'+ ' &lt;&lt;');	
+	}
+}
+function submitFilter() {
+	document.aimTeamAuditListForm.submit();
+}
+function resetSearch() {
+	document.getElementById("userId").selectedIndex = 0;
+	document.getElementById("teamId").selectedIndex = 0;
+	document.getElementById("selectedDateFromText").value="";
+	document.getElementById("selectedDateToText").value="";
+    document.aimAuditLoggerManagerForm.action = "/aim/teamAuditList.do?action=reset";
+    document.aimTeamAuditListForm.submit();
+}
+
+
 </script>
+
+<jsp:include page="/repository/aim/view/scripts/newCalendar.jsp"  />
+<style>
+.contentbox_border{
+	border:1px solid #666666;
+	width:100%;	
+	background-color: #f4f4f2;
+	padding: 20 0 20 0;
+}
+</style>
+<style>
+.link{
+	text-decoration: none;
+	font-size: 8pt; font-family: Tahoma;
+}
+</style>
+
+<style>
+
+.tableEven {
+	background-color:#dbe5f1;
+	font-size:8pt;
+	padding:2px;
+}
+
+.tableOdd {
+	background-color:#FFFFFF;
+	font-size:8pt;!important
+	padding:2px;
+}
+ 
+.Hovered {
+	background-color:#a5bcf2;
+}
+.divTable{
+	display: table;
+	width: 100%;
+}
+.divTableRow {
+	display: table-row;
+}
+.divTableCellLeft{
+	text-align: right;
+	font-weight: bold;
+}
+.divTableCell {
+	display: table-cell;
+	padding: 3px 10px;
+	width: 50%;
+}
+.divTableBody {

```

Comment:
the css code and jsp code is already present in audit logger view, we should reuse using jsp includes and js script files